index.js

```diff
@@ -21,56 +14,40 @@ module.exports = forwarded
  * @public
  */
 
-function forwarded (req) {
+module.exports = function forwarded (req) {
   if (!req) {
     throw new TypeError('argument req is required')
   }
 
   // simple header parsing
-  const proxyAddrs = parse(req.headers['x-forwarded-for'] || '')
   const socketAddr = req.socket.remoteAddress
```

Comment:
Maybe it gets some yoctoseconds faster if i dont use the variable and directly use it in the code below.